### ISSUE TITLE
Request for 'read whole line till enter' and send

### ISSUE THREAD

**Issue by kutysam** [pyserial/pyserial#200]
I'm currently having issues communicating with a bluetooth module AT-09 with serial.
It works if i do the following in a .py file and import serial
'command = input('Enter your command: ')
ser.write(str.encode(command + "\r\n"))
data = ser.readline()'

My command is 'AT+POWE'
However, in miniterm, the momment i type 'A' the output gives me an ERR.
So i tried ser.write(str.encode("A")) in python and it also gave me ERR

My best bet is, miniterm sends out anything you type immediately.
The bluetooth module is expecting a full command within like 0.1seconds?
I tried copying and pasting AT+POWE in miniterm but it doesn't work still. I guess I'm too slow or something.

However, i believe with ser.write(str.encode(command + "\r\n")) done in codes in python itself, python is able to communicate in blazing fast speed such that the whole command gets send within 0.01seconds or something.

It will be nice if minicom has a flag that allows us to send a full command after pressing enter instead of sending every character immediately after it's typed.

**Comment by zsquareplusc** [comment]
Yes miniterm sends the characters as you type. The idea is to keep miniterm simple but reliable to test connections, so i'd rather not add such a buffering feature which needs configuration. But sending and receiving in chunks is also easily done in a interactive Python session.

For AT commands, see also [examples/at_protocol.py](https://github.com/pyserial/pyserial/blob/master/examples/at_protocol.py)
